lib/puppet/file_bucket/file.rb

```diff
@@ -79,6 +78,10 @@ def self.from_s(contents)
     self.new(contents)
   end
 
+  def self.from_binary(contents)
+    self.new(contents)
+  end
+
```

Comment:
Cool, I also reworked `.to_s` to directly call `to_binary` to make it a little more obvious